Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -403,6 +403,50 @@ instance IccChartedSpace (x y : ℝ) [h : Fact (x < y)] :
       simpa only [not_lt] using h'
   chart_mem_atlas z := by by_cases h' : (z : ℝ) < y <;> simp [h']
 
+@[simp]
+lemma iccManifold_chartAt {x y : ℝ} [h : Fact (x < y)] {z : Set.Icc x y} :
```

Comment:
```suggestion
lemma IccManifold_chartAt {x y : ℝ} [h : Fact (x < y)] {z : Set.Icc x y} :
```
Same below

Actually, I'm not sure what the `Manifold` is referring to here.

Reply:
Good point; `Manifold` should be `ChartedSpace`. (I'm not sure if that's a great name - better suggestions welcome.)
Would `chartAt_iccChartedSpace` be better?

Just to check: why should it be UpperCamelCase? I would have thought `snake_case` (rules 1 and 5).

Reply:
So `Manifold`/`ChartedSpace` comes from `ChartedSpace.chartAt`? If so, the name should rather be
```suggestion
lemma Icc_chartedSpaceChartAt {x y : ℝ} [h : Fact (x < y)] {z : Set.Icc x y} :
```
or
```suggestion
lemma chartedSpaceChartAt_Icc {x y : ℝ} [h : Fact (x < y)] {z : Set.Icc x y} :
```
or some namespaced version thereof.

`Icc` (as well as the other intervals) is an exception to the rules.

Reply:
Thanks for explaining. I've gone with `Icc_chartedSpaceChartAt` and similar.